.github/CODEOWNERS

```diff
@@ -22,10 +22,10 @@
 /docs/source-pytorch/index.rst              @williamfalcon @lantiga
 /docs/source-pytorch/levels                 @williamfalcon @RobertLaurella
 /docs/source-pytorch/expertise_levels       @williamfalcon @RobertLaurella
-/docs/source-app/                           @williamfalcon @RobertLaurella @tchaton @awaelchli
-/docs/source-app/conf.py                    @borda @awaelchli @carmocca
-/docs/source-app/index.rst                  @williamfalcon @lantiga
-/docs/source-app/expertise_levels           @williamfalcon @RobertLaurella
+/docs/source-lit/                           @williamfalcon @RobertLaurella @tchaton @awaelchli
```

Comment:
can we call this source-lightning instead?

Reply:
or simply `docs/source/`?

Reply:
source is fine :)